wgsl/index.bs

```diff
@@ -1309,57 +1309,142 @@ repeatedly replacing a subsequence with its homoglyph.)
 
 A <dfn>context-dependent name</dfn> is a [=token=] used to name a concept, but only in specific grammatical contexts.
 The spelling of the token may be the same as an [=identifier=], but the token does not [=resolve=] to a declared object.
+This section lists the tokens used as [=context-dependent names=].
```

Comment:
I would add:
```suggestion
This section lists the tokens used as [=context-dependent names=].
The token must not be a [=keyword=] or [=reserved word=]. See [[#builtin-inputs-outputs]].
```

Then remove all the identifier resolution repetition from subsequent subsections:
* Built-in value names
* Diagnostic rule names
* Interpolation type names
* Interpolation sampling names

Reply:
Good one, thank you!